src/main/resources/avro/referencemethods.avdl

```diff
@@ -261,7 +260,69 @@ ListReferenceBasesResponse getReferenceBases(
   /** Additional request parameters to restrict the query. */
   ListReferenceBasesRequest request) throws GAException;
 
-/****************  /sequence/{id}  *******************/
+/******************  /sequences/search  *********************/
+/**
+This request maps to the body of `POST /sequences/search` as JSON. Specifies a
+number of filters, all of which must be satisfird by each result returned.
+*/
+record SearchSequencesRequest {
+  /**
+  If not null, return only `Sequence`s that appear in the indicated
```

Comment:
We should clarify that exactly one of referenceSetId or variantSetId is required to be non-null.